docs/software/overview.md

```diff
@@ -0,0 +1,63 @@
+---
+tags:
+ - Software
+---
+# Software Overview
+
+Each ORCD system has its own software stack. Many basic and commonly used software and libraries are already installed, so it is good to check before spending the time to install it yourself. This page discusses the general overview of what kinds of software are supported and points you to how to use them and what to do if what you need isn't there.
+
+## Software Landscape
+
+While the software stack will be different on each system, there are three general classes of software:
```

Comment:
@lmilechin what about using a table here e.g.

The standard software stack is slightly different on each generation of ORCD system. Software packages on a system always fall into one of three general categories outlined below:

| Category  | Description                                                                                                                        |
|-----------|------------------------------------------------------------------------------------------------------------------------------------|
| _Core_      | Commonly used or fundamental software and libraries that are fully supported. Core software is expected to work until it is officially deprecated, and often newer versions are provided to replace them. |
| _Community_ | Software that has been built and installed by request, but is not commonly used. Support is on a best-effort basis. Should work when built but not guaranteed to work indefinitely or when replaced with newer versions when deprecated, except by request. |
| _Deprecated_ | Software that is no longer supported or expected to work. May be kept for legacy reasons, or will soon be removed. If software you are using is listed as deprecated or soon to be deprecated migrate to the newer version (if available) or request a newer version (if not available). If migrating to a newer version is not an option you may be able to run your application with Singularity. |

Reply:
Looks good! I've changed it to a table